core/src/main/java/org/apache/gravitino/stats/StatisticManager.java

```diff
@@ -58,7 +58,7 @@
 
 public class StatisticManager implements Closeable {
 
-  private static final String OPTIONS_PREFIX = "gravitino.stats.partition.option.";
+  private static final String OPTIONS_PREFIX = "gravitino.partition.stats.";
```

Comment:
I think it is better to change to `gravitino.stats.partition.*` to align with "gravitino.stats.partition.storageFactoryClass".

Reply:
With this, you need to introduce another configuration like `gravitino.stats.partition.storage` = `lance` to specify the storage name.

Reply:
It's a little complex. We only have one storage factory.

Reply:
OK.